### PR TITLE
restricts user

### PR DESCRIPTION
**Pull request by himanshujaidka**
## Related Issuse

- Info about Issue or bug

Closes: #308 [issue number that will be closed through this PR]
@codewithvk pls help

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.(Specially for Backend)
- [ ] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Review by codewithvk — CHANGES_REQUESTED** [review]
Can you please tell me where is middleware condition?

File: server/src/common-middleware/middleware.js [hunk 1]
```diff
@@ -23,7 +23,18 @@ const userMiddleWare = (req, res, next) => {
     next();
 };
 
+const spaceMiddleware = (req, res, next) => {
+    if(req.user.role == "user"){
```
Comment (codewithvk) [inline server/src/common-middleware/middleware.js:27]: why are you check this one?

**Comment by himanshujaidka** [comment]
@codewithvk any updates

**Comment by codewithvk** [comment]
@himanshujaidka This might be not working as we think, Do you have any other approach!?

**Comment by himanshujaidka** [comment]
@codewithvk will let you know

**Comment by himanshujaidka** [comment]
@codewithvk tell me that how should I will able to increase the counter to plus one automatically after the creation of space. That when the user will create space, then in its respected database, the counter should increase to 1. And for this first, we should declare counter inside spaces models file and then after the creation of space, it would be increased to 1. Did you get what I am able to deliver?

**Pull request closed without merging**